File: Pak Dengklek dan Deret yang Indah/Pak-Dengklek-dan-Deret-yang-Indah.c
#include <stdio.h>

int main ()
{
    short S = 0, N = 0, D = 0, mulai = 0;
    scanf ("%hd %hd %hd", &S, &N, &D);

    mulai = S;

    for (short i = 0; i < N; i++)
    {
        printf ("%hd\n", mulai);
        mulai += D;
    }

    return 0;
}

File: Pak Dengklek dan Deret yang Indah/PakDengklekdanDeretyangIndah.java
import java.util.Scanner;

public class PakDengklekdanDeretyangIndah
{
    public static void main (String [] args)
    {
        Scanner scan = new Scanner (System.in);
        short S = scan.nextShort ();
        short N = scan.nextShort ();
        short D = scan.nextShort ();
        scan.close ();

        short mulai = S;
        for (short i = 0; i < N; i++)
        {
            System.out.println (mulai);
            mulai += D;
        }
    }    
}